common/lib/dependabot/clients/azure.rb

```diff
@@ -190,6 +190,33 @@ def create_pull_request(pr_name, source_branch, target_branch,
           "/pullrequests?api-version=5.0", content.to_json)
       end
 
+      # rubocop:disable Metrics/ParameterLists
+      def autocomplete_pull_request(pull_request_id, auto_complete_set_by, merge_commit_message,
+                                    delete_source_branch = true, squash_merge = true, merge_strategy = "squash",
```

Comment:
Did you test that `merge_strategy = "squash"` works? 

Per https://github.com/dependabot/dependabot-core/pull/4064#issuecomment-1251896060 it needs to be a quoted number, but if they support passing the string `"squash"` let's stick with what you've got as it's certainly more readable!

Reply:
Yes, we have this as part of our `generic-update-script.rb`: https://github.com/aviationexam/dependabot-script/blob/avi/main/generic-update-script.rb#L476 and it works well

Reply:
Reading the doc: https://learn.microsoft.com/en-us/rest/api/azure/devops/git/pull-requests/update?view=azure-devops-rest-5.1&tabs=HTTP#gitpullrequestmergestrategy is seemed like it should be a string

Reply:
Good. Will migrate too.

Reply:
Thanks for testing @trejjam 👍